hw02/EnemyCollision.cs

```diff
@@ -18,15 +18,23 @@ void Update()
     
     // TODO: Problem 7: Why did I not include the words public or private here?
     // What does it mean when I only write void (Movement)? What does void mean?
+    
+    //public means you can access it globally,while private you can not. if no modifier is provided, it defalts to private
+    //void is a return type, it means the function doesn't return a value.
     void Movement ()
     {
         // TODO: Problem 8: Please explain what the next 4 lines mean.
         // What is Input.GetAxis, transform.Translate, and transform.Rotate? 
         float forwardMovement = Input.GetAxis("Vertical") * speed * Time.deltaTime;
+        //detects forward and backward movement
         float turnMovement = Input.GetAxis("Horizontal") * turnSpeed * Time.deltaTime;
-        
+        //detects turning movement
         transform.Translate(Vector3.forward * forwardMovement);
+        //move the player forward/backward depending on input and speed
```

Comment:
After we get the inputs and speed at a consistent speed (Time.deltaTime) we then want to transform. There we use Unity's `Translate`. Here we are using vectors to move our player according to our input and speed.